provisioning/resources/ui/js/components/ControlPlane.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+/// <reference path="../../typings/node/node.d.ts" />
+/// <reference path="../../typings/react/react.d.ts" />
+/// <reference path="../../typings/react/react-dom.d.ts" />
+/// <reference path=".././Interfaces.d.ts"/>
+
+import React = require("react");
+import ReactDOM = require("react-dom");
+import axios from 'axios';
+
+export class ControlPlane extends React.Component<{}, {}> {
+
+  public render() {
+    return (
+      <div className="tab-content">
+        <p>Press the buttons for controlling the Snowplow-Mini without ssh:<br></br><br></br>
```

Comment:
> Press the buttons for controlling the Snowplow-Mini without ssh:

This feels like we are oversharing what it is doing.  Why does the user care that it is without ssh?

Can we change this to:

> The buttons below can be used to interact with the interact with the internal systems of Snowplow Mini:

Thoughts @alexanderdean ?